R/plotStorms.R

```diff
@@ -2,6 +2,42 @@
 
 
 
+#' Get type for icon (leaflet)
+#'
+#' @noRd
+#' @param msw numeric
+#'
+#' @return corresponding icon
+getType <- function(msw) {
```

Comment:
I think this function is not used. Needs to be removed

Reply:
Done, I forgot indeed